packages/nextjs/components/Spinner.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
This is how I see it (no animation). 

![image](https://user-images.githubusercontent.com/2486142/206681527-717c8ef4-5c65-49b4-93f3-d65dcd0dd6c0.png)

Maybe you see it correctly in the forced dark mode?

Also the "X" it's too close. No big deal for now tho.

Reply:
In https://github.com/scaffold-eth/se-2/pull/55/commits/041925bd7d24f1d5c30719125d8393f9422ef973 I have just updated the color, could please check it once again? 
For me it's working 
Light : 
![Screenshot 2022-12-09 at 5 24 47 PM](https://user-images.githubusercontent.com/80153681/206697733-62ed7d70-e889-41aa-a39c-a31837883c03.jpg)
Dark : 
![Screenshot 2022-12-09 at 5 24 10 PM](https://user-images.githubusercontent.com/80153681/206697762-71ade07a-d483-444f-82e3-97452dacc504.jpg)


PS: Even I had the same problem at the very start, the in-between animation was not showing, but it was solved once I changed the color of  `fill-color` in `Spinner.jsx`. Please let me know if the problem still exist (The spinner code was take from [here](https://flowbite.com/docs/components/spinner/))

Reply:
TW is not compiling the filling color for me (since it's dynamic).

https://stackoverflow.com/a/71068925

This is why I did this:

![image](https://user-images.githubusercontent.com/2486142/207014992-42107e73-e731-499e-9b68-3c9ca122a37e.png)

Not sure why it's working for you :D 

Anyway, I hardcoded it for now: 64a3624e5b9bc8596fe092211cd584dbde34b195